include/SFML/Window/WindowBase.hpp

```diff
@@ -424,6 +424,44 @@ class SFML_WINDOW_API WindowBase
     ////////////////////////////////////////////////////////////
     void setIcon(Vector2u size, const std::uint8_t* pixels);
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the window state
+    ///
+    /// Change between windowed and fullscreen mode.
```

Comment:
This phrasing is brittle. Once we add more states as we have planned, this comment will go out of date. Can we use more generic language that doesn't require mentioning the windowed and fullscreen states?